stable/fluent-bit/templates/daemonset.yaml

```diff
@@ -78,11 +87,10 @@ spec:
           mountPath: /fluent-bit/etc/parsers_custom.conf
           subPath: parsers.conf
 {{- end }}
-{{- end }}
```

Comment:
This seems broken here... where did the end go? its merging a large chunk of code into the above conditional incorrectly?